docs/tutorials.mdx

```diff
@@ -117,4 +117,12 @@ These tutorials are intended for developers who are familiar with Tezos and want
   link="Start tutorial"
 />
 
+<TutorialCard
+  title="Running the DAL as a Weeklynet baker, in 5 steps"
```

Comment:
Title reads weird as we don't run the DAL (it entails we somehow own it?) but rather join/participate in it with a baker?

Reply:
```suggestion
  title="Joining the Tezos DAL as a Weeklynet baker, in 5 steps"
```

Reply:
Maybe even remove the mention of Weeklynet, which will be replaced by other testnets when protocol 0 goes out:
```suggestion
  title="Running the DAL as a baker, in 5 steps"
```

Reply:
How about "Running a baker on the DAL?"

Reply:
In f8ac5c1 I went for "How to join the Tezos DAL as baker, in 5 steps".